docs/release.md

```diff
@@ -0,0 +1,26 @@
+# Release Process
+This document describes how to cut a `cri` plugin release.
+
+## Step 1: Update containerd vendor
+Before cutting a release, update the containerd vendor to latest
+version in the corresponding branch, and make sure all tests in
+CI pass https://k8s-testgrid.appspot.com/sig-node-containerd.
+## Step 2: Cut the release
+Draft a new release in https://github.com/containerd/cri/releases.
+## Step 3: Update `cri` version in containerd
+Send a PR to containerd to update `cri` vendor version to the
```

Comment:
Push a PR to containerd/containerd that updates the version of containerd/cri in containerd/containerd/vendor.conf to the newly tagged release created in Step 2.

Reply:
Done